src/core/dbus.c

```diff
@@ -1144,12 +1145,12 @@ int bus_fdset_add_all(Manager *m, FDSet *fds) {
         return 0;
 }
 
-int bus_foreach_bus(
-                Manager *m,
-                sd_bus_track *subscribed2,
-                int (*send_message)(sd_bus *bus, void *userdata),
-                void *userdata) {
-
+static int bus_foreach_bus_internal(Manager *m,
+                                    sd_bus_track *subscribed,
+                                    bool only_with_subscribers,
+                                    int (*send_message)(sd_bus *bus, void *userdata),
+                                    void *userdata)
+{
```

Comment:
Please read https://systemd.io/CODING_STYLE/ first. Specifically, it's correct to break line immediately after `(`, and `{` should follow `)`